apps/src/applab/designMode.js

```diff
@@ -547,6 +548,24 @@ function duplicateScreen(element) {
   return newScreen;
 }
 
+designMode.onCopyElementToScreen = function (element, destScreen) {
+  const sourceScreen = $(element);
```

Comment:
Maybe we should rename `sourceScreen` to something like `sourceElement`, since it won't always be a screen. (In fact, will it ever be a screen?)

Reply:
Done.